Nie-ontologies/Generic-ontologies/publishing.ttl

```diff
@@ -599,7 +599,7 @@ publish:hasPrinterDescriptionLiteral
 publish:isSerialIssueOfVolume
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/publishing#>;
 	a owl:ObjectProperty;
-	rdfs:label "is serial of volume"@en, "ist Reihe-Ausgabe eines Volumens"@de, "est publication sérielle du volume"@fr, "è pubblicazione seriale del volume"@it;
+	rdfs:label "is serial of volume"@en, "ist Serie eines Bandes"@de, "est publication sérielle du volume"@fr, "è pubblicazione seriale del volume"@it;
```

Comment:
is serial of volume -> is serial issue of volume; ist Serie eines Bandes -> ist Serienausgabe eines Bandes ?

Reply:
Serienausgabe eines Bandes. Corrected it.

Reply:
Commit a901c55